config/channels/in-memory-channel/200-channelable-clusterrole.yaml

```diff
@@ -0,0 +1,33 @@
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: imc-channelable
+  labels:
+    duck.knative.dev/channelable: "true"
```

Comment:
@Harwayne I'm sure you're already thinking this, but this should also make it's way into the Channel Spec.

Reply:
Done.